lib/generators/react_on_rails/templates/Procfile.dev.server

```diff
@@ -0,0 +1,4 @@
+web: rails s
+client: sh -c 'rm app/assets/javascripts/generated/* || true && cd client && npm run build:dev:client'
+server: sh -c 'cd client && npm run build:dev:server'
```

Comment:
I'd rather we dry this up, put hot after web, and then append the server line if needed.